lucene/spatial3d/src/test/org/apache/lucene/spatial3d/TestGeo3DPoint.java

```diff
@@ -84,6 +85,10 @@
 
 public class TestGeo3DPoint extends LuceneTestCase {
 
+  protected PlanetModel randomPlanetModel() {
+    return RandomPicks.randomFrom(random(), new PlanetModel[] {/*PlanetModel.WGS84,*/ PlanetModel.CLARKE_1866});
+  }
```

Comment:
Can we add the three planet models in the random?

Reply:
+1